README.md

```diff
@@ -142,7 +161,12 @@ Add a precommit hook to `package.json` to automatically lint and format any file
 
 If you've added Prettier to an existing project you will want to format all the code. The precommit hook only updates files that have been changed and staged for commit. To format the entire codebase run
 
-`yarn format`
+`npm run format`
+
+## Upgrading to version 7
+
+- `npm install -D eslint@latest eslint-config-neo@latest`
```

Comment:
We should try and make these upgrade instructions more generic. Also not sure we really ever want to use the `latest` tag. I'd suggest using `npm upgrade` or something like `npm-check-updates` instead.